lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "lerna": "2.10.2",
+  "lerna": "2.11.0",
```

Comment:
Was the update of lerna necessary to get ga working?

Reply:
No – just taking the opportunity to get our dependencies updated. Will be more mindful about separating out the PRs next time.